src/algebra/ordered_ring.lean

```diff
@@ -807,8 +796,8 @@ namespace nonneg_ring
 open nonneg_add_comm_group
 variable [nonneg_ring α]
 
-@[priority 100] -- see Note [lower instance priority]
-instance to_ordered_ring : ordered_ring α :=
+local attribute [instance]
```

Comment:
```suggestion
/-- Construct `ordered_ring` from `nonneg_ring`. This is not an instance
because it is not used in `mathlib`. -/
local attribute [instance]
```